cli/main.go

```diff
@@ -98,6 +101,8 @@ func main() {
 		instOptions = append(instOptions, auto.WithGlobal())
 	}
 
+	instOptions = append(instOptions, auto.WithLogLevel(logLevel))
```

Comment:
```suggestion
	if logLevel != "" {
		instOptions = append(instOptions, auto.WithLogLevel(logLevel))
	}
```

This relates to my other comment in this file

Reply:
Changed